README.md

```diff
@@ -1,27 +1,4 @@
 <a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
```

Comment:
- [ ] Kindly make sure you use the correct GitHub flow.
![image](https://user-images.githubusercontent.com/97100091/204556408-9535d499-a181-4b50-aae2-cbc17634cdf9.png)
The Pull request is supposed to be `into main form feature_branch` and not the other way around
- [ ] Also consider giving your PR a descriptive title telling what the project is about.
`hint: Hello-microverse` sound better than `asking for pull request` :face_with_spiral_eyes: